automatic/activepresenter/tools/chocolateyInstall.ps1

```diff
@@ -1,13 +1,23 @@
 $ErrorActionPreference = 'Stop'
 
+$fileType      = 'exe'
+$toolsDir      = Split-Path $MyInvocation.MyCommand.Definition
+$embedded_path = gi "$toolsDir\*.$fileType"
```

Comment:
Now the only thing that troubled me is that it it changes installer file name. This was never problematic but if you actually donwload software which includes x64 and x32 this will add just one of those to the end which might be confusing to somebody looking into the package after unpacking it - one may think its x32 or x64 while its actualy both.

I decided so far that this issue does not deserves attention.